civihr_default_theme/scss/components/_button.scss

```diff
@@ -35,3 +35,16 @@
 .btn-xs {
   @include button-size($padding-xs-vertical, $padding-xs-horizontal, $font-size-small, $line-height-small, $border-radius-small);
 }
+
+.btn-gradient {
+  @include button-size($padding-large-vertical, $padding-large-horizontal, $font-size-base, $line-height-large, $border-radius-large);
+
+  background: $chr-gradient-1;
+  border: 0;
+  color: $light;
+
+  &:hover {
+    background: $chr-gradient-2;
```

Comment:
i thought you were going to darken/lighten it? Defining a gradient just for a hover state seems a bit overkill

Reply:
We should keep the variables, otherwise there is a chance of using different darken/lighten percentage values. I have changed the code to define the variables though
```scss
$chr-primary-gradient-stop-1-color:    #01a2d5;
$chr-primary-gradient-stop-2-color:    #0070bb;
$chr-primary-gradient:                 linear-gradient(315deg, $chr-primary-gradient-stop-1-color 0%, $chr-primary-gradient-stop-2-color 100%);
$chr-primary-gradient-hover:           linear-gradient(315deg, darken($chr-primary-gradient-stop-1-color, 7%) 0%, darken($chr-primary-gradient-stop-2-color, 7%) 100%);
```